### ISSUE TITLE
Multiple pact-directories

### ISSUE THREAD

**Issue by DerKnecht** [DiUS/pact-jvm#652]
I would like to have a feature to specify more than one pact directory.
We have projects with subprojects and when building with jenkins the path differs from the path we use when executing locally. So the need appears to define more than one directory to search pacts in it...
Locally I have to use _`<pactDirectory>target/pacts</pactDirectory>`_
But with Jenkins it would be _`<pactDirectory>mysubproject/target/pacts</pactDirectory>`_.

**Comment by uglyog** [comment]
Would this be for finding pacts to verify, or different directories that the pacts are written to?

**Comment by DerKnecht** [comment]
This would be for finding pacts to verify.

**Comment by uglyog** [comment]
That should be easy to implement. Would you want it to find all pacts within a directory tree, or just specific sub-directories?

**Comment by DerKnecht** [comment]
It would be enough for me to specify directories like this:
```
<pactDirectories>
   <pactDirectory>target/pacts</pactDirectory
   <pactDirectory>mysubproject/target/pacts</pactDirectory>
</pactDirectories>
```

**Comment by DerKnecht** [comment]
Additionally it would be great if when a directory is not found, that one is just skipped with no effect (the others should be checked after).

**Comment by uglyog** [comment]
Fix has been released with version 3.5.18
